conf_collectd_plugin.py

```diff
@@ -0,0 +1,375 @@
+import sys
```

Comment:
any script which might be invoked directly (as opposed to imported) needs a shebang line.

you probably want "#!/usr/bin/env python"

dont forget to make your scripts run on 2.7 or 3.0 by writing portable code, and using the new style print statements. http://stackoverflow.com/questions/32032697/how-to-use-from-future-import-print-function